pkg/registries/registries.go

```diff
@@ -101,6 +101,12 @@ func extractRegistryHostname(fullImage string) string {
 	return strings.Split(fullImage, imagePathSeparator)[0]
 }
 
+// extractOrgPath removes the registry hostname from the given image
+func extractOrgPath(image string) string {
+	hostname := extractRegistryHostname(image)
+	return strings.TrimPrefix(strings.SplitN(image, hostname, 2)[1], "/")
```

Comment:
Should we add a check for empty hostname to avoid panic here?

Reply:
Added